workers_kv_example_test.go

```diff
@@ -0,0 +1,137 @@
+package cloudflare_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+)
+
+const (
+	namespace = "xxxxxx96ee002e8xxxxxx665354c0449"
+	org       = "xxxxxx10ee002e8xxxxxx665354c0410"
+)
+
+func ExampleAPI_CreateStorageNamespace() {
+	api, err := cloudflare.New(apiKey, user, cloudflare.UsingOrganization(org))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	req := &cloudflare.StorageNamespaceRequest{Title: "test_namespace2"}
+	snr, err := api.CreateStorageNamespace(context.Background(), req)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Println(snr)
+}
+
+func ExampleAPI_ListStorageNamespaces() {
+	api, err := cloudflare.New(apiKey, user, cloudflare.UsingOrganization(org))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	lsr, err := api.ListStorageNamespaces(context.Background())
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Println(lsr)
+}
+
+func ExampleAPI_DeleteStorageNamespace() {
+	api, err := cloudflare.New(apiKey, user, cloudflare.UsingOrganization(org))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	response, err := api.DeleteStorageNamespace(context.Background(), namespace)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Println(response)
+}
+
+func ExampleAPI_UpdateStorageNamespace() {
+	api, err := cloudflare.New(apiKey, user, cloudflare.UsingOrganization(org))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	resp, err := api.UpdateStorageNamespace(context.Background(), namespace, &cloudflare.StorageNamespaceRequest{Title: "test_title"})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Println(resp)
+}
+
+func ExampleAPI_CreateStorageKV() {
+	api, err := cloudflare.New(apiKey, user, cloudflare.UsingOrganization(org))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	payload := strings.NewReader("test payload")
+	key := "test_key"
+
+	resp, err := api.CreateStorageKV(context.Background(), namespace, key, payload)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if payload.Len() > 0 {
+		log.Fatalf("Reader was not drained, remaining bytes: %d", payload.Len())
+	}
```

Comment:
is this necessary? shouldn't this be reported in `CreateStorageKV`'s error?

Reply:
This is an example test. We don't have to check anything here it was just for debugging. I can remove.

Reply:
>  shouldn't this be reported in CreateStorageKV's error?

I don't think so because that's the underlying http client's responsibility. I'm open to suggestions if you disagree.